Domain.Sql/EventExtensions.cs

```diff
@@ -55,26 +55,32 @@ public static StorableEvent ToStorableEvent(this IEvent domainEvent)
         /// Creates a <see cref="StorableEvent" /> based on the specified domain event.
         /// </summary>
         /// <param name="domainEvent">The domain event.</param>
-        internal static StorableEvent ToStorableEvent<TAggregate>(this IEvent<TAggregate> domainEvent)
-            where TAggregate : IEventSourced =>
-                new StorableEvent
-                {
-                    Actor = domainEvent.Actor(),
-                    StreamName = AggregateType<TAggregate>.EventStreamName,
-                    SequenceNumber = domainEvent.SequenceNumber,
-                    AggregateId = domainEvent.AggregateId,
-                    Type = domainEvent.EventName(),
-                    Body = JsonConvert.SerializeObject(domainEvent, Formatting.None, serializerSettings.Value),
-                    Timestamp = domainEvent.Timestamp,
-                    ETag = domainEvent.ETag
-                };
+        /// <param name="serializationFunc">Optional func to call to serialize the domain event.</param>
+        internal static StorableEvent ToStorableEvent<TAggregate>(this IEvent<TAggregate> domainEvent, SerializeEvent serializationFunc)
+            where TAggregate : IEventSourced
```

Comment:
Typically delegate parameters or variables are named for the action they perform, so a verb instead of a noun: `serialize`.

Reply:
will fix!

Reply:
will fix everywhere serializationFunc and deserializationFunc are used

Reply:
Isn't serialization the noun?

Reply:
Oh you mean the variable name, not the delegate name... n/m